qr_generator/lib/data/sqlite_persistence.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:path/path.dart';
+import 'package:sqflite/sqflite.dart';
+import 'package:sqflite/sqlite_api.dart';
+
+class SqlitePersistence {
```

Comment:
**question:** I haven't seen a SQL implementation like this in quite some bit. What made you choose to use this approach versus other available alternatives in the community?

Reply:
Hi Jeff, thank you for your feedback. 
About my choice of implementation, I based my choice on an example I found in Medium. I'll look into other approaches for this type of implementation. If you have any specific suggestions, I'd much appreciate them.